diannayim/Resume
# feat: Adding github workflow to publish webpage

### Description
* Initialized NextJS Typescript project
* Added github workflow files

### Ticket 
#1

### Testing
1. Run the application.
2. Check that the project is built to the correct path in Github using Github Actions.

### Follow-up Required
* Make sure to change the branch name back to main after testing.
